packages/events/src/router/event/index.ts

```diff
@@ -139,9 +141,21 @@ export const eventRouter = router({
     }),
   draft: router({
     list: publicProcedure.output(z.array(Draft)).query(async (options) => {
+      if (options.ctx.userType === TokenUserType.SYSTEM) {
+        throw new TRPCError({
+          code: 'NOT_IMPLEMENTED',
+          message: 'Drafts are not supported for system users'
+        })
+      }
       return getDraftsByUserId(options.ctx.user.id)
     }),
     create: publicProcedure.input(DraftInput).mutation(async (options) => {
+      if (options.ctx.userType === TokenUserType.SYSTEM) {
+        throw new TRPCError({
+          code: 'NOT_IMPLEMENTED',
+          message: 'Drafts are not supported for system users'
+        })
+      }
```

Comment:
could this be like `.use(notForSystemUsers)` ?

Reply:
if the Context type gets updated after using that, then it could be a better option. Otherwise let's stick with the current one